File: Solved/02577/02577.cpp
// Problem No.: 2577
// Solver:      Jinmin Goh
// Date:        20191127
// URL: https://www.acmicpc.net/problem/2577

#include <iostream>
#include <list>
#include <stdlib.h>
using namespace std;

int main()
{
    // this code makes input & output fast as scanf / printf
    // cin.tie(NULL);
    // ios_base::sync_with_stdio(false);
    // check following blog: https://starrykss.tistory.com/750

    int a, b, c, num;
    cin >> a >> b >> c;
    num = a * b * c;

    int count[11] = {0};

    while(num != 0)
    {
        count[num % 10]++;
        num /= 10;
    }

    for(int i = 0; i < 10; i++)
        cout << count[i] << endl;

    return 0;
}

File: Solved/02577/02577.py
# Problem No.: 2577
# Solver:      Jinmin Goh
# Date:        20191127
# URL: https://www.acmicpc.net/problem/2577

import sys

a = int(input())
b = int(input())
c = int(input())
num = a * b * c
count = [0,0,0,0,0,0,0,0,0,0]

while num != 0:
    count[num % 10] += 1
    num = num // 10

for i in range(10):
    print(count[i])